microservices/google-api/google.googlesearch.sql

```diff
@@ -0,0 +1,15 @@
+DROP TABLE IF EXISTS google.googlesearch;
+CREATE TABLE IF NOT EXISTS google.googlesearch (
+	site    VARCHAR(255)    ENCODE ZSTD,
+	date    date,
+	query   VARCHAR(2047)    ENCODE ZSTD,
+	country VARCHAR(255)    ENCODE ZSTD,
+	device  VARCHAR(255)    ENCODE ZSTD,
+	page    VARCHAR(2047)    ENCODE ZSTD,
+	position FLOAT,
+	clicks   DECIMAL,
```

Comment:
Why decimal instead of integer?

Reply:
It was initially committed that way:

https://github.com/bcgov/GDX-Analytics/commit/d4929906c1cdb86a1f38b9f3558d8a86b7546d0b#diff-9a9c0f3bd6854e88ad0eeaf10e67c1f1

Reply:
However, I don't know if there's any reason why it shouldn't be an integer..

Reply:
We should probably change it. I think?

Not sure though. I'll leave it up to you.

Reply:
@danpollock I'm not too sure either. It works, but there is some additional storage expense to it ([redshift docs](https://docs.aws.amazon.com/redshift/latest/dg/r_Numeric_types201.html#r_Numeric_types201-decimal-or-numeric-type)). I can change it to `INTEGER`, but it would need to be tomorrow.

Reply:
Sounds good.

Reply:
Click and impressions were updated to be INTEGER instead of DECIMAL. This is more efficient. I have resolved this thread.